salt/modules/zypper.py

```diff
@@ -1080,16 +1098,41 @@ def upgrade(refresh=True):
     .. code-block:: bash
 
         salt '*' pkg.upgrade
+        salt '*' pkg.upgrade dist-upgrade=True fromrepo="MyRepoName" novendorchange=True
+        salt '*' pkg.upgrade dist-upgrade=True dryrun=True
     '''
     ret = {'changes': {},
            'result': True,
            'comment': '',
     }
 
+    cmd_update = ['dist-upgrade'] if dist_upgrade else ['update']
+    cmd_update.extend(['--auto-agree-with-licenses'])
```

Comment:
No need in extra brackets and it can be shorter. This is sufficient:
```python
cmd_update = (['dist-upgrade'] if dist_upgrade else ['update']) + ['--auto-agree-with-licenses']
```